hack/swagger-doc/build.gradle

```diff
@@ -0,0 +1,18 @@
+buildscript {
+    repositories {
+        mavenLocal()
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'io.github.robwin:swagger2markup:0.6.0'
```

Comment:
Ugg, you want to run Java as part of the build?  As if it's not slow enough.  We have no precedent for this, so I don't even know what questions to ask.  What JVM is required?  Where do I download this module?  Do we need to vendor it?